src/components/report-technical-issue/template.njk

```diff
@@ -0,0 +1,2 @@
+<a class="govuk-link {{ params.classList }}" target="_blank" href="/contact/problem_reports_nonjs?service={{ params.serviceCode | default("unknown") }}"
+>{% if params.language == 'cy' %}Help gyda'r dudalen hon{% else %}Get help with this page{% endif %}</a>
```

Comment:
Is there any reason you've chosen this wording as opposed to some of the options in services such as 'Is there anything wrong with this page?' or 'Is this page not working properly?'.?

Reply:
The reason is to encourage this conversation. If there's an "approved" new set of wording I'd love to include it.

Reply:
The default should be changed to a blank string.

Reply:
Add query parameter `newTab=true` - this will be ignored but will allow us to update the messaging at a later date.